### DISCUSSION TITLE
DSiWare still crashing

### DISCUSSION THREAD

**Discussion by ariebe9115** [rafaelvcaetano/melonDS-android#726] [Q&A]
Just wanted to ask if DSiWare will ever be supported or if you gotta fix the other things first, i'll attach a video showing an instantly crashing DSiWare which isnt the devices fault since SnapDragon 860 should be fine, i think this is happening because of Microphone access but i dont know, does anyone know why this is happening? 

https://user-images.githubusercontent.com/64021242/142737169-f38b34ac-9b86-49d0-b80b-a15a1aee56a2.mp4

**Comment by Damaj301damaj-lol** [comment]
Dsi is still experimental, and melonDS has full support for microphones, i tested it in a compatible game and it worked
